src/theme/bootstrap-override/bootstrap-italia/_type.scss

```diff
@@ -1,10 +1,13 @@
 html,
 body.public-ui {
-  font-size: 16px;
-  line-height: 1.5;
+  // font-size: 16px;  ? Already set by bootstrap-italia
+  // line-height: 1.5; ? Already set by bootstrap-italia
 
   @include media-breakpoint-up(sm) {
     font-size: 18px;
-    line-height: 1.555;
+    // line-height: 1.555; ? Already set by bootstrap-italia
+  }
+  blockquote:before {
```

Comment:
questa regola sui blockquote siamo sicuri di volerla mettere in questo file? non c'è un file dedicato?